src/sos-packaging.props

```diff
@@ -10,7 +10,9 @@
   <ItemGroup>
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\sos.dll" TargetRid="win-x64" />
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\Microsoft.DiaSymReader.Native.amd64.dll" TargetRid="win-x64" />
+  </ItemGroup>
       
+  <ItemGroup Condition="'$(BuildX64Package)' != 'true'">
```

Comment:
Why is this condition needed?

Reply:
This allows me to build just the x64 packages locally so I can reference them in the internal repo for testing.  Normally in the official builds we need all the platform and architecture versions to build packages.  It is a short cut for local development.